OrbitQt/orbitcodeeditor.cpp

```diff
@@ -61,7 +61,6 @@
 #include <QtWidgets>
 #include <fstream>
 
-#include "App.h"
```

Comment:
Maybe we should remove the complete class (maybe in a different PR). WDYT?

Reply:
Yes, this will be revamped with the source code view. So there are already a variety of bugs covering that.

Reply:
Fine. Thanks.